### PR TITLE
fix swap/savings rap types

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes TEAM2-202

## What changed (plus any additional context for devs)
mapping between swap modal types & rap types was not set up correctly. I added some logic to change before we submit estimations + raps since I don't want to mess with the swap modal types

## PoW (screenshots / screen recordings)
https://cloud.skylarbarrera.com/Screen-Recording-2022-06-28-13-30-15.mp4

## Dev checklist for QA: what to test

withdraw + savings should work now

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [x] Added e2e tests? if not please specify why
- [x] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

File: src/screens/ExchangeModal.js [hunk 1]
```diff
@@ -259,7 +260,22 @@ export default function ExchangeModal({
...
+        case ExchangeModalTypes.withdrawal:
+          rapType = RapActionTypes.withdrawCompound;
+          break;
+        case ExchangeModalTypes.deposit:
+          rapType = RapActionTypes.depositCompound;
+          break;
+        default:
+          rapType = RapActionTypes.swap;
+          break;
+      }
```
[the remark this answers is not in the document]
Comment (estebanmino) [reply src/screens/ExchangeModal.js:274]: just to clarify when i say "the same" is that the results are the same but now we have more lines of code

File: src/screens/ExchangeModal.js [hunk 2]
```diff
@@ -420,7 +436,19 @@ export default function ExchangeModal({
...
```
Comment (estebanmino) [inline src/screens/ExchangeModal.js:450]: wdyt of creating another method `getSwapRapTypeByExchangeType` containing this switch case so you can use it [here](https://github.com/rainbow-me/rainbow/pull/3613/files#diff-08d9008ab79a0765a3f4bca960079242e26c4a35125cf214a538f43497d9fd95R264)  as well

**Review by ibrahimtaveras00 — APPROVED** [review]
Fix looks good on both OS's, QA Passed 👍🏽

**Review by estebanmino — APPROVED** [review]
didi a deposit in hardhat, looking good!

**Pull request merged**